_pages/home/updating-to-version-4/updating-to-version-4.md

```diff
@@ -257,6 +257,9 @@ ng update @spartacus/schematics@4
 
 When the update has finished running, inspect your code for comments that begin with `// TODO:Spartacus`. For detailed information about each added comment, see the [Detailed List of Changes]({{ site.baseurl }}/technical-changes-version-4/#detailed-list-of-changes).
 
+### Import `AppRoutingModule` from `@spartacus/storefront` instead of local module
+After bumping to 4.0 you can drop your local file `app-routing.module`. Instead of importing the local `AppRoutingModule` in your `AppModule`, please import the `AppRoutingModule` from `@spartacus/storefront`.
```

Comment:
Maybe we could phrase it like this:
```suggestion
After bumping to 4.0 you can drop your local file `app-routing.module` (if it exists). Instead of importing the local `AppRoutingModule` in your `AppModule`, please import the `AppRoutingModule` from `@spartacus/storefront`.
```